src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx

```diff
@@ -50,10 +52,10 @@ const ListItem = styled.div`
   padding: ${({ theme }) => theme.spaces.xs} 0 ${({ theme }) => theme.spaces.xs}
     ${({ theme }) => theme.spaces.xs};
   cursor: pointer;
+`;
 
-  div {
-    margin: 0 0 0 ${({ theme }) => theme.spaces.xs};
-  }
+const ListLabel = styled.div`
```

Comment:
I think this should be a `p`.

Reply:
OK fixed 9944530 I did the same for `<MobileNavigationBar />`